packages/ws-worker/test/lightning.test.ts

```diff
@@ -444,7 +447,85 @@ test('should not claim while at capacity', async (t) => {
   });
 });
 
-// hmm, i don't even think I can test this in the mock runtime
-test.skip('should pass the right dataclip when running in parallel', () => {});
+test('should pass the right dataclip when running in parallel', (t) => {
```

Comment:
As a result of using the new runtime, I can now write really sophisticated tests like this, which deeply test a) how the runtime handles complex Attempts and b) how the worker maps runtime events to Lightning events.

If I had this test two weeks ago, I'd have skipped over a lot of hurt this week!

This proves the value of this work, in the next couple of weeks I'll be building more more complex tests like this.